docs/index.md

```diff
@@ -1,30 +1,48 @@
+for `kaniko-project/executor`. It reduces the complexity of building Docker images and removes the need of using Docker
+in Docker when building images in K8s infrastructure.
 
-[Image Detector](/development/image-detector/README.md) - Image Detector is a tool for updating the security scanner config with the list of images in the Prow cluster. To achieve that, it receives paths to files used to deploy Prow or its components.
+[Image Detector](/cmd/image-detector/README.md) - Image Detector is a tool for updating the security scanner config with
+the list of images in the Prow cluster. To achieve that, it receives paths to files used to deploy Prow or its
+components.
 
-[image-syncer](/development/image-syncer/README.md) - image-syncer is used to copy container images from one registry to another.
+[image-syncer](/cmd/image-syncer/README.md) - image-syncer is used to copy container images from one registry to
+another.
 
-[Image URL Helper](/development/image-url-helper/README.md) - Image URL Helper is a tool that provides the following subcommands:
+[Image URL Helper](/cmd/image-url-helper/README.md) - Image URL Helper is a tool that provides the following
+subcommands:
 
-[JobGuard ](/development/jobguard/README.md) - JobGuard is a simple tool that fetches all statuses for GitHub pull requests and waits for some of them to finish.
+[prowjobparser](/cmd/prowjobparser/README.md) - The prowjobparser is a helper tool which parses all Prow Jobs under the
+provided path, matches them against the provided label filters, and prints matching Prow Job names to the standard
+output.
 
-[prowjobparser](/development/prowjobparser/README.md) - The prowjobparser is a helper tool which parses all Prow Jobs under the provided path, matches them against the provided label filters, and prints matching Prow Job names to the standard output.
+[Rotate service account secrets](/cmd/cloud-run/rotate-service-account/README.md) - RotateServiceAccount creates a new
+key for a GCP service account and updates the required secret data. It's triggered by a Pub/Sub message sent by a secret
+stored in Secret Manager. It runs as a cloud run container.
 
-[Rotate service account secrets](/development/secrets-rotator/cloud-run/rotate-service-account/README.md) - RotateServiceAccount creates a new key for a GCP service account and updates the required secret data. It's triggered by a  Pub/Sub message sent by a secret stored in Secret Manager. It runs as a cloud run container.
-
-[Cleanup of service account secrets](/development/secrets-rotator/cloud-run/service-account-keys-cleaner/README.md) - The Cloud Run service deletes old keys for a GCP service account and updates the required secret data for all service account secrets stored in the Secret Manager. The service is triggered by a Cloud Scheduler job.
+[Cleanup of service account secrets](/cmd/cloud-run/service-account-keys-cleaner/README.md) - The Cloud Run service
```

Comment:
Throughout the documentation, the word 'secret' is used multiple times, and it's always spelled with lowercase 's'. Please remember to [capitalize 'Secret' whenever you refer to a Kubernetes resource](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalization).